### PR TITLE
fix cross compile on target aarch64-linux-android

### PR DESCRIPTION
**Pull request by chshawkn**
fix https://github.com/carllerche/mio/issues/598

### CODE REVIEW

**Comment by carllerche** [comment]
Thanks... do you know if it would be possible to get this build somehow integrated in CI? To avoid breaking this in the future...

**Comment by chshawkn** [comment]
I surrender, can't get CI work...

**Comment by MortimerGoro** [comment]
Hello, I'm working on adding aarch64 support for Servo and this compilation error is one of the blockers. Could you move the CI integration to a separate PR to avoid delaying the compilation error fix?

File: Cargo.toml [hunk 1]
```diff
@@ -26,7 +26,7 @@ default = ["with-deprecated"]
 lazycell = "0.4.0"
 log      = "0.3.1"
 slab     = "0.3.0"
-net2     = "0.2.19"
+net2     = "0.2.29"
```
Comment (carllerche) [inline Cargo.toml]: Is bumping this version required to get aarch64-linux-android to pass?

Comment (chshawkn) [inline Cargo.toml]: Yes, I sent a similar PR to net2-rs few days ago.

**Comment by chshawkn** [comment]
@MortimerGoro I will do it later

**Comment by carllerche** [comment]
This was merged into master as part of #601.

**Pull request closed without merging**